cli/cmd/devtool/data/cloud-deps.docker-compose.yml

```diff
@@ -31,14 +31,14 @@ services:
   otel-collector:
     image: otel/opentelemetry-collector:latest
     restart: always
-    command: ["--config=/etc/otel-collector-config.yaml"]
+    command: [ "--config=/etc/otel-collector-config.yaml" ]
     volumes:
       - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
     ports:
-      - "1888:1888"   # pprof extension
-      - "8889:8889"   # Prometheus exporter metrics
+      - "1888:1888" # pprof extension
+      - "8889:8889" # Prometheus exporter metrics
       - "13133:13133" # health_check extension
-      - "4317:4317"   # OTLP gRPC receiver
+      - "4317:4317" # OTLP gRPC receiver
       - "55679:55679" # zpages extension
```

Comment:
Revert changes here (since it's not related to this PR)

Reply:
this is done by linter run i guess, and if revert it i.e remove spaces from `"1888:1888" # pprof extension` in this show compile time error